src/app/authorization/login/login.component.ts

```diff
@@ -46,8 +44,8 @@ export class LoginComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.createForm();
-    if (localStorage.getItem('registeruser')?.length) {
-      let data: any = localStorage.getItem('registeruser');
+    if (localStorage.getItem('registerUser')?.length) {
```

Comment:
no need to check length as localstorage will return stringnot array

Reply:
ok, i removed it.